### PR TITLE
refactor #627 moving members to python-gitlab

### PR DESCRIPTION
**Pull request by Shackelford-Arden**
Moving `members` over for #627

edit: Fixed issue link.

### CODE REVIEW

File: gitlabform/processors/project/members_processor.py [hunk 1]
```diff
@@ -101,11 +102,25 @@ def _process_users(
         enforce_members: bool,
         keep_bots: bool,
     ):
+
+        project: Project = self.gl.projects.get(project_and_group)
```
Comment (TimKnight-DWP) [inline gitlabform/processors/project/members_processor.py:106]: is there a reason not using the cached method here?

**Review by TimKnight-DWP — CHANGES_REQUESTED** [review]
One small query - otherwise looks good, especially that none of the service tests needed to be change to support this 👍

File: gitlabform/processors/project/members_processor.py (hunk 1 above)
Comment (Shackelford-Arden) [reply gitlabform/processors/project/members_processor.py:106]: Mostly that I wasn't fully aware of it 😁 I've stepped away for a bit, but will pull in the cached method instead, as that would definitely be better!

Comment (Shackelford-Arden) [reply gitlabform/processors/project/members_processor.py]: Updated :)

**Comment by TimKnight-DWP** [comment]
I might get chance to merge today - but I won't get chance to make a new release til next week

**Comment by TimKnight-DWP** [comment]
Unlikely to merge today - stuff come up at the day job 👍

**Comment by TimKnight-DWP** [comment]
@Shackelford-Arden had a few incidents to handle last week but I _should_ have capacity today 👍

**Comment by Shackelford-Arden** [comment]
> @Shackelford-Arden had a few incidents to handle last week but I _should_ have capacity today 👍 

No worries 🙂 Thank you!

**Pull request merged**